resolwe/flow/models/entity.py

```diff
@@ -26,3 +27,39 @@ class Meta(BaseCollection.Meta):
 
     #: indicate whether `descriptor` is completed (set by user)
     descriptor_completed = models.BooleanField(default=False)
+
+
+class RelationType(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+
+    ordered = models.BooleanField(default=False)
```

Comment:
Idea is that client (or package with defined relation types) takes care of ordering. 
`position` can be something like `0`, `1`, `beginning`, `intermediate `, `end`, `case`, `control`,... So there is no general rule for ordering.